WalletWasabi/BitcoinCore/Monitoring/P2pReconnector.cs

```diff
@@ -7,52 +7,46 @@
 
 namespace WalletWasabi.BitcoinCore.Monitoring
 {
-	public class P2pReconnector : PeriodicRunner
-	{
-		public P2pReconnector(TimeSpan period, P2pNode p2pNode) : base(period)
-		{
-			P2pNode = Guard.NotNull(nameof(p2pNode), p2pNode);
-			Success = new TaskCompletionSource<object>();
-		}
+    public class P2pReconnector : PeriodicRunner
```

Comment:
![image](https://user-images.githubusercontent.com/9156103/99074241-b0c6e600-25b7-11eb-8151-2455a19ae8a7.png)

Reply:
must be a hotkey that I pressed accidentally, my bad